sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -361,5 +363,28 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
         return eglConfigOk
       }
     }
+
+    /**
+     * Static method to check if [MapView] could properly render 3D terrain on this device.
+     * This method may take some time on slow devices.
+     *
+     * @return true if 3D terrain could be rendered on this device and false otherwise
+     */
+    @JvmStatic
+    fun isTerrainRenderingSupported(): Boolean {
```

Comment:
nit:
if it static method, maybe move to some extension file. `MapView.kt` is already to big